Cargo.toml

```diff
@@ -49,3 +50,6 @@ tracing = { version = "0.1", default-features = false }
 tracing-core = { version = "0.1", default-features = false }
 tracing-subscriber = { version = "0.3", default-features = false }
 url = { version = "2.2", default-features = false }
+
+[patch.crates-io]
+tonic = { git = "https://github.com/hyperium/tonic.git", rev = "9b306af386528f11dbd022bc372d367adc4e96b5" }
```

Comment:
can you add a comment why is this required.

Reply:
Added